server.js

```diff
@@ -16,7 +17,13 @@ const transport = new Parity.Api.Transport.Http(`http://localhost:${config.get('
 const api = new Parity.Api(transport);
 
 const app = express();
+// Support health checking by sending HEAD
+app.head('/', (req, res) => res.status(200).end());
```

Comment:
Maybe also the Parity node status ? Like block number/timestamp... ?

Reply:
Makes sense. Added `/health` endpoint.